controllers/storageoscluster/node-manager_operand.go

```diff
@@ -222,45 +222,32 @@ func (c *NodeManagerOperand) Ensure(ctx context.Context, obj client.Object, owne
 		return nil, err
 	}
 
-	cluster, ok := obj.(*storageoscomv1.StorageOSCluster)
-	if !ok {
-		return nil, fmt.Errorf("failed to convert %v to StorageOSCluster", obj)
-	}
-
 	currState := c.getCurrentState()
-	if len(cluster.Spec.NodeManagerFeatures) > 0 && !currState {
+	if !currState {
```

Comment:
This means node manager became a permanent feature. I'm not really sure about this. Should we discuss about his team wide?

Reply:
Will be discussed in slack